service/src/main/java/org/databiosphere/workspacedataservice/dao/RecordDao.java

```
@@ -190,6 +195,7 @@ public void batchUpsertWithErrorCapture(UUID instanceId, RecordType recordType,
 					throw new BatchWriteException(rowErrors);
 				}
 			}
+			throw e;
```

Also swallowing exceptions here.